cli/src/main/java/org/infinispan/cli/completers/AuthorizationPermissionCompleter.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.cli.completers;
+
+/**
+ * @since 14.0
+ */
+public class AuthorizationPermissionCompleter extends EnumCompleter<AuthorizationPermissionCompleter.AuthorizationPermission> {
+   public enum AuthorizationPermission {
```

Comment:
we have an enum in `core`. Can't we move it to `commons`?

Reply:
No, it's not useful for Hot Rod.